src/styles/LandingPage.module.less

```diff
@@ -37,17 +38,44 @@
     right: 0;
     left: 0;
     width: 75.3vw;
-    color: #373737;
+    color: #ffffff;
     font-size: 3.644rem; 
-    font-weight: 900;
+    font-weight: 400;
     text-align: left;
-    line-height: 7.822rem;
-    padding-left: 5%;
+    line-height: 7.157rem;
+    letter-spacing: 0.1rem;
+    padding-left: 0%;
 }
 
 // Use strong to give first word emphasis 
 .LandingPageOverlayText strong{ 
-    color: #0b688a;
+    color: #ffffff;
+    font-size: 4.063rem; 
+    font-weight: bold;
+}
+
+.LandingPageOverlayText button{ 
+    background-color: #3b9ec2;
+    line-height: 2.15;
+    text-align: center;
+    font-size: 1.7rem; 
+    font-weight: bold;
+    color: #fff;
+    border: 0;
+    border-radius: 3rem;
+    width: 26rem;
+    height: 4.79542334076rem;
+    padding-top: 0.5rem;
```

Comment:
This is the source of the issue. Probably just remove it or make it the same for the buttom.

Reply:
Just the `padding-top` part to be exact.